src/lv_draw/lv_draw_arc.c

```diff
@@ -107,8 +107,8 @@ static void get_rounded_area(int16_t angle, lv_coord_t radius, uint8_t tickness,
     const uint8_t ps = 8;
     const uint8_t pa = 127;
 
-    lv_coord_t thick_half = tickness / 2;
-    lv_coord_t thick_corr = tickness & 0x01 ? 0 : 1;
+    int32_t thick_half = tickness / 2;
+    int32_t thick_corr = tickness & 0x01 ? 0 : 1;
```

Comment:
Actually a boolean would be big enough. :-)
Suggest to use an uint8_t instead.

Reply:
I agree; `thick_corr` should be a `uint8_t`.